docs/content/concepts/layer/validation/_index.md

```diff
@@ -0,0 +1,61 @@
+aliases: []
+suppresstopiclist: true
+---
+
+## Who is this document for?
+
+If you are an [OSCAL tool developer](https://pages.nist.gov/OSCAL/about/stakeholders/tooldevelopers/), you should read this document to understand OSCAL, including core terminology the OSCAL community uses in development and troubleshooting.
+
+If you are not an OSCAL tool developer, this technical information might be useful either if you seek general understanding of OSCAL, or if as a tool *user* you encounter them in your work.
+
+## Why are these concepts important?
+
+It is important that OSCAL tool developers implement software that confirm the JSON, XML, and YAML is well-formed and the [OSCAL document instances](https://pages.nist.gov/OSCAL/concepts/layer/) are valid.
+
+Generally, both "validity" and "well-formedness" define OSCAL, such that a conformant OSCAL application can respect them, and an application that does not is *ipso facto* not an OSCAL application.
+
+## What do "well-formed" and "valid" mean to the OSCAL community?
+
+When developing tools to read or write OSCAL, NIST or fellow community developers will ask if your data is *well-formed* and if the OSCAL document is *valid*. We define these terms below and explain how current OSCAL tools support checking well-formed data and valid OSCAL.
+
+### What is well-formed data?
+
+*Well-formed* means that your JSON, XML, YAML (whether it is OSCAL or not) follows the syntax rules of that format.
+
+#### What OSCAL tools help you check well-formedness?
+
+NIST's OSCAL tools do not explicitly check that JSON, XML, or YAML is well-formed. If you are a tool developer, we expect you to use your language runtime and libraries to parse JSON, XML, and YAML and check for the well-formed data. This must occur before validating an OSCAL document.
+
+### What is a valid OSCAL document?
+
+*Valid* means that your OSCAL document (in a JSON, XML, or YAML already checked for *well-formedness*) follows the rules of a model. A model can be expressed using a schema format, such as [JSON Schema](https://json-schema.org/) or [XML Schema](https://www.w3.org/XML/Schema).
+
+### Does OSCAL provide schemas for OSCAL models?
+
+The NIST OSCAL Team develops and maintains [JSON Schema](https://github.com/usnistgov/OSCAL/tree/main/json/schema) to validate OSCAL documents in JSON and YAML data formats and [XML Schema](https://github.com/usnistgov/OSCAL/tree/main/xml/schema) to validate OSCAL documents in the XML data format.
+
+
+### What tools check that an OSCAL document is valid?
+
+There are many tools to validate documents using schema formats.
+
+Here are some Internet resources to help you find a tool.
+
+- [JSON Schema validator](https://json-schema.org/implementations.html#validators)
+- [XML Schema validators](https://www.w3.org/XML/Schema#Tools)
+
+**Note: These lists are not exaustive and NIST does not endorse any of these tools.**
+
+### Why can OSCAL document validation fail?
```

Comment:
This section seems to end abruptly. We talk about failure modes, but say nothing about how to address them. Not sure what to do about this, since we don't want to make this document much larger.

Reply:
I agree, so one of the remaining items with regards to this PR would be referencing to the "how to validate OSCAL?" content we agree to split into a different page, and later just a README in usnistgov/OSCAL#1170.

Would a link to this other Markdown README in the repo saying "for further explanation of how validation occurs and how to react to different failure modes, see this document" here be enough or you are thinking something more comprehensive?

Reply:
This is deep b/c failure mode and appropriate remediation depends on scope of authority. Can the data be corrected, remediated or otherwise flagged, or must it be pushed back?

For one use case, e.g. editing content, to be able intervene and adjust to make invalid content into valid content is a requirement; to another (e.g., receiving data as a "blind" submission) the rule must be "no correction, only rejection".

Additionally, not all validation rules will have the same impacts, operationally, and they need to be addressed differently depending on how the data is produced.

AJ have the old standards been played yet: "when producing, be strict; when consuming, be amenable"?

Reply:
Apologies, I spun off....

Reply:
I didn't think you spun off, I thought it was important and relevant to the discussion here. It might just go to another part of another document later.

I talked with Dave earlier in the afternoon and he recommended we purse approach 1 with additional caveat: for each one of the failure conditions in the list, add a brief sentence about the approach you would use to debug/resolve it further. After we comfortable with #1170, we will add the link in this page to that resulting GitHub link to that file.